### ISSUE TITLE
"Error: private repository" on Releases pages

### ISSUE THREAD

**Issue by kapsh** [ovity/octotree#743]
### Description

Octotree complains that repository is private on every Releases page. It makes no sense since I don't have tokens for all repos anyway.

Random example: https://github.com/sharkdp/fd/releases

### Environment (if bug)

* Octotree version: 3.0.8
* Browser & version: Chrome 75.0.3770.142
* OS & version: Linux
* Screenshot, if any (drag an image here)
![image](https://user-images.githubusercontent.com/144717/61554834-8c600580-aa66-11e9-9dc8-94f004660823.png)

* Console error log below this line
```
content.js:2117 TypeError: Cannot read property 'addEventListener' of null
    at forked_to_init (content.js:6733)
    at features_run (content.js:2112)
    at content.js:2158
    at content.js:2073
    at onAjaxedPagesRaw (content.js:2068)
features_run @ content.js:2117
async function (async)
features_run @ content.js:2112
(anonymous) @ content.js:2158
(anonymous) @ content.js:2073
onAjaxedPagesRaw @ content.js:2068
async function (async)
onAjaxedPagesRaw @ content.js:2066
onAjaxedPages @ content.js:2071
add @ content.js:2158
async function (async)
add @ content.js:2124
(anonymous) @ content.js:6747
__webpack_require__ @ content.js:12
(anonymous) @ content.js:64
(anonymous) @ content.js:65
content.js:2116 ❌ release-download-count
content.js:2117 Error: Personal token required for this feature
    at content.js:2306
    at async getAssetsForTag (content.js:6456)
    at async release_download_count_init (content.js:6484)
    at async features_run (content.js:2112)
```

**Comment by jdreesen** [comment]
I had the same problem once. But now it works, also with your link, although I did not (consciously) change anything.

For the reference: #732

**Comment by kapsh** [comment]
@jdreesen hello! Thanks for quick answer. I like Octotree and I am opened for cooperation if it will help make it better. Please tell what additional infromation should I collect.

**Comment by stevennoto** [comment]
I was able to reproduce this; in my case it was related to the extension [Refined GitHub](https://github.com/sindresorhus/refined-github). I think in my case at least that extension messed with Octotree's detection of which branch name to load, which then caused the API call to fail. I will see if I can fix this.

@kapsh, may I ask if you have that extension (or any other GitHub-page-altering extensions) installed?

**Comment by kapsh** [comment]
Good catch, I have Refined github installed. My bad I didn't think about possible conflicts. 
When that extension is disabled, Octotree works correct on Releases pages.

**Comment by jdreesen** [comment]
I've refined GitHub installed, too. Sadly, octotree isn't playing very well with other GitHub extensions on general :/

**Comment by stevennoto** [comment]
Cool, that means we can identify the issue and fix it! I will look into this.

**Comment by stevennoto** [comment]
I have fixed this in #811. The problem was that Refined Github reuses the branch-picker dropdown for other things in the releases page, confusing Octotree's branch detection. My PR avoids this by defaulting to `master` on these pages, and the tree now loads properly for me. :)
